website/docs/FAQ.md

```diff
@@ -127,3 +127,34 @@ Workaround:
 2. `mkdir /home/vscode/.local/lib/python3.10/site-packages/google/colab`
 
 Explanation: Per [this gist](https://gist.github.com/defulmere/8b9695e415a44271061cc8e272f3c300?permalink_comment_id=4711478#gistcomment-4711478), linked from the official [chromadb docs](https://docs.trychroma.com/troubleshooting#sqlite), adding this folder triggers chromadb to use pysqlite3 instead of the default.
+
+## How to register a reply function
+
+(from [issue #478](https://github.com/microsoft/autogen/issues/478))
+
+You can hook in a function that is called whenever an agent receives a message
+
+Assuming you have two agents user_proxy and assistant, you can register a function with these agents that gets called with some parameters.
```

Comment:
"you can register a function with these agents that gets called with some parameters" is a bit too generic on its own. Perhaps link to the documentation: https://microsoft.github.io/autogen/docs/reference/agentchat/conversable_agent#register_reply

Reply:
Thanks @afourney

Reply:
I agree with adding a link to the documentation.